packages/dx-react-chart-demos/src/demo-sources/export/material-ui/demo.jsx

```diff
@@ -0,0 +1,297 @@
+import React, { useState } from 'react';
+import Paper from '@material-ui/core/Paper';
+import {
+  ArgumentAxis,
+  ValueAxis,
+  Chart,
+  Legend,
+  BarSeries,
+  Title,
+} from '@devexpress/dx-react-chart-material-ui';
+import { Plugin, Template, TemplatePlaceholder } from '@devexpress/dx-react-core';
+import domtoimage from 'dom-to-image';
+import JsPDF from 'jspdf';
+import IconButton from '@material-ui/core/IconButton';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import { scaleBand } from '@devexpress/dx-chart-core';
+import { ArgumentScale, Stack, Animation } from '@devexpress/dx-react-chart';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+
+import { gaming as data } from '../../../demo-data/data-vizualization';
+
+const options = [
+  'Print',
+  'PNG',
+  'JPEG',
+  'PDF',
+];
+
+const styles = () => ({
+  button: {
+    width: '50px',
+    height: '50px',
+  },
+});
+
+const addKeyframe = (name, def) => {
+  if (typeof document === 'undefined') {
+    return;
+  }
+  const head = document.getElementsByTagName('head')[0];
+  let style = Array.from(head.getElementsByTagName('style'))
+    .find(node => node.dataset[Symbol('animation')]);
+  if (!style) {
+    style = document.createElement('style');
+    style.type = 'text/css';
+    style.dataset[Symbol('animation')] = true;
+    head.appendChild(style);
+  }
+  const content = style.textContent;
+  if (!content.includes(name)) {
+    style.textContent += `\n@keyframes ${name} ${def}\n`;
+  }
+};
+
+const getLabelAnimationName = () => {
```

Comment:
Wouldn't it be simpler to turn animation off rather then having such code?

Reply:
It is be very strange if only one demo will be without animation. And it is useful to show users how they can animate labels.

Reply:
OK.